core/securitykey.py

```diff
@@ -1,33 +1,36 @@
         that key. These are usually used as a CSRF token.
         This has been changed from ViUR2 - where it was possible to create a arbitrary number of security keys per
         session.
-    - If :meth:create is called with a duration (and optional keyword-parameters), it will create a security key
-        that is *not* bound to the current session, but it's possible to store custom data (the excess keyword
-        arguments passed to :meth:create). As these are not bound to the session, each call to :meth:create will yield
-        a new token. These are used if it's expected that the token may be redeemed on a different device (eg. when
-        sending an email address confirmation email)
-
-    ..note: There's a hidden 3rd type of security-key: The sessions static security key. This key is only revealed once
-        (during login, as the protected header Sec-X-ViUR-StaticSKey). This can be used instead of the onetime sessions
-        security key by sending it back as the same protected http header and setting the skey value to
-        "staticSessionKey". This is only intended for non-webbrowser, programmatic access
-        (ViUR Admin, import tools etc) where CSRF attacks are not applicable. Therefore that header is prefixed with
-        "Sec-" - so it cannot be read or set by javascript.
+    2. If :meth:create is called with a duration (and optional kwargs values), it will create a security key
+        that is *not* bound to the current session, but it's possible to store custom data (provided by kwargs).
+        As these are not bound to the session, each call to :meth:create will yield a new token.
+        These are used if it's expected that the token may be redeemed on a different device (e.g. when sending an
+        email address confirmation)
+
+    ..note:
+        There's also a hidden 3rd type of security-key: The sessions static security key.
+
+        This key is only revealed once (during login, as the protected header Sec-X-ViUR-StaticSKey).
+
+        This can be used instead of the one-time sessions security key by sending it back as the same protected HTTP
+        header and setting the skey value to "staticSessionKey". This is only intended for non-web-browser,
+        programmatic access (admin tools, import tools etc.) where CSRF attacks are not applicable.
+
+        Therefor that header is prefixed with "Sec-" - so it cannot be read or set using JavaScript.
 """
+from viur.core import utils, current, db, tasks
 from datetime import datetime, timedelta
-from viur.core.utils import generateRandomString
-from viur.core import current, db
-from viur.core.tasks import PeriodicTask, CallDeferred
 from typing import Union
-from viur.core.utils import utcNow
 
-securityKeyKindName = "viur-securitykeys"
+SECURITYKEY_KINDNAME = "viur-securitykeys"
 
 
 def create(duration: Union[None, int] = None, **kwargs) -> str:
```

Comment:
```suggestion
def create(duration: Union[None, int] = None, **custom_data) -> str:
```
and ff. 

to clarify that these are not parameters passed to a super call but individual data.